eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -366,22 +375,38 @@ function DeleteOrUpdateResourceGroups() {
     if ((IsChildResource $rg) -or (HasDeleteLock $rg)) {
       continue
     }
-    if ((HasDoNotDeleteTag $rg) -or (HasValidAliasInName $rg) -or (HasValidOwnerTag $rg)) {
+    if (HasDoNotDeleteTag $rg) {
```

Comment:
Do we still want to support this tag? Is there a way to put a lock on a resource group?

Reply:
Yes you can add a delete lock. Let's chat more on the Monday meeting about this. It could be nice to remove the tag, though a benefit is we know we added it manually vs. a delete lock that may have been part of a deployment or some other automatic reason.